packages/cli/src/commands/generate/scaffold/__tests__/fixtures/components/groupForm.js

```diff
@@ -0,0 +1,40 @@
+import {
```

Comment:
this is my stripped down test file for form, is this the best way to go about this?